docs/docs/general/03-Element Selection/aielement.md

```diff
@@ -130,3 +130,46 @@ AskUI-RemoveAIElement -NameFilter "logo" -AutoApproval
 # Opens all AI Elements with the name 'logo'
 AskUI-OpenAIElement -NameFilter "logo"
 ```
+
+## Using AI Elements with Git
+
+AI Elements can be version controlled and shared across team members using Git. Here's how to set it up:
+
+### Current Behavior
+By default, AI Elements are stored in the Global AI Element Location under `~/.askui/SnippingTool/AIElement/<workspace-id>/`. This location is not version controlled by Git.
+
+### Prerequisites
+- Update askui to version 0.22.0 or higher:
+```bash
+npm install askui@latest
+```
+
+### Configuration Steps
+1. Create a folder in your Git project called `<project>/ai-elements`
+2. Update your `askui_example/helpers/askui-helper.ts`:
+
+```typescript
+beforeAll(async () => {
+  aui = await UiControlClient.build({
+    // ... existing configuration ...
+    aiElementArgs: {
+      additionalLocations: [
+        "./ai-elements/" // Add your local AI Element location here
+      ]
+    }
+  });
+});
+```
```

Comment:
I would add more of the `// ... existing configuration ...` around the code to indicate everywhere where there may be other code and point out that the user has to find wherever `UiControlClient.build` is called and add it there